conf/machine/qcm6490-idp.conf

```diff
@@ -9,3 +9,7 @@ MACHINE_FEATURES = "usbhost usbgadget alsa wifi bluetooth"
 KERNEL_DEVICETREE ?= " \
                       qcom/qcm6490-idp.dtb \
                       "
+
+MACHINE_ESSENTIAL_EXTRA_RRECOMMENDS += " \
```

Comment:
Technically this is not an essential package to install as part of the image, since it is used together with the image while flashing the device, probably better to use EXTRA_IMAGEDEPENDS instead.

```
MACHINE_ESSENTIAL_EXTRA_RDEPENDS[doc] = "A list of required machine-specific packages to install as part of the image being built. Because this is a 'machine essential' variable, the list of packages are essential for the machine to boot."
```

Reply:
@ricardosalveti without these boot loader firmware files being flashed the device cannot get to a bootable state. 
Isn't that a reason that requires these to be included in MACHINE_ESSENTIAL_EXTRA_RRECOMMENDS?

Reply:
if it's in MACHINE_ESSENTIAL_EXTRA_RRECOMMENDS, then it will be added into the target image, which we don't need/want. EXTRA_IMAGEDEPENDS is for recipes needed to build/deploy, but they don't get added to the image.

Reply:
Right, the difference is that here we don't need/want the boot firmware to be installed as part of the image, but to be built along with it.